lms/static/styles/_utils.scss

```diff
@@ -19,3 +19,17 @@
 .u-cross-stretch {
   align-self: stretch;
 }
+
+// Constrain and overflow with ellipsis
+.u-overflow--ellipsis {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+// A columnar flex layout centered on the cross axis
+.u-layout-column--align-center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
The idea here is to corral reusable utility-like classes for later evaluation for migration to the `frontend-shared` package (where they would be prefixed `.hyp-u-`). I've got some documentation about all of this started; hope to be able to share next week. Lots going on!